src/content/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial.mdx

```diff
@@ -233,6 +244,10 @@ Queries make requests to fetch data. To learn additional query capabilities avai
 
 Below are some sample requests to automate the creation of your synthetic monitors. 
 
+<Callout variant="info">
```

Comment:
Also, what color do you want this? For a tip, use this code:

```
<Callout variant="tip">
text here
</Callout>
```

for important:

```
<Callout variant="important">
text here
</Callout>
```

Reply:
I think using a tip makes sense.